content/influxdb/cloud/notebooks/create-notebook.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Create a notebook
+description:
+weight: 102
+influxdb/v2.0/tags:
+menu:
+  influxdb_2_0:
+    name: Create a notebook
+    parent: Notebooks
+---
+{{% note %}}
+**Notebooks is currently an early access feature.**
+[Submit a request](https://w2.influxdata.com/notebooks-early-access/ ) to be added to the queue, and we will send you a confirmation when you’ve been added to early access.
+{{% /note %}}
+
+Create a notebook to explore, visualize, and process your data.
+
+This guide walks through the basics of creating a notebook. For specific examples, see the following:
+  - [Downsample data](/influxdb/cloud/notebooks/downsample/)
+  - [Normalize data](/influxdb/cloud/notebooks/normalize/)
+
+## Create a new notebook
+1. In the navigation menu on the left, click the **Notebooks**.
+
+    {{< nav-icon "notebooks" >}}
+2. Click **+Create Notebook**.
+3. Enter a name for your notebook in the **Name this notebook** field.
+4. By default, a **Metric Selector** and **Visualization** cell appear (see [Cell types](#cell-types) below for details.)
+5. Click the **+** icon to add a cell. See [Cell types](#cell-types) below for details on each type of cell.
+6. Click **Preview** to preview the results of each cell in a raw data table without writing any data.
+7. Click **Run** to show the results of each cell and write it to the specified output bucket.
+7. Click the eye icon to hide a cell.
+
+## Cell types
+Notebooks are comprised of different cells. The following cell types are available:
+- **Input: Metric Selector**:
+  - Select a bucket to define your data source.
+  - Edit your time range with the time range option in the dropdown menu.
+  - Add filters to narrow your data.
+- **Transform: Flux Script**:
+  - Use `__PREVIOUS_RESULT__` to build from data in the previous cell.
+  - Enter a Flux script to transform your data.
+- **Pass-through: Markdown**: Enter explanatory notes or other information for your team members in Markdown.
```

Comment:
"you or your team members" ? you can just leave notes to yourself and since the vast majority of our users are single member orgs... not sure